crates/db_views/src/post_view.rs

```diff
@@ -170,6 +176,7 @@ pub struct PostQueryBuilder<'a> {
   show_bot_accounts: Option<bool>,
   show_read_posts: Option<bool>,
   saved_only: Option<bool>,
+  languages: Option<Vec<LanguageId>>,
```

Comment:
Languages isn't necessary, because you only need `my_person_id`, which can join to the `local_user_languages` table.